visualization/app/codeCharta/ui/dialogs/changelogDialog/changelogDialog.component.ts

```diff
@@ -11,7 +11,8 @@ export class ChangelogDialogComponent {
 	changes: Record<string, string>
 
 	constructor(@Inject(MAT_DIALOG_DATA) public data: { previousVersion: string; currentVersion: string }) {
-		let changelogLines = markdownFile.split("\n")
+		const newLineStartPattern = /(?<=>)\s+(?=<)/g
```

Comment:
I am not certain if the regex is correct. I checked it by regex101. Can you please confirm this?

Reply:
We should indeed split the lines similar to before. I guess we might also remove potential carriage returns.